aten/src/ATen/core/MT19937RNGEngine.h

```diff
@@ -93,28 +93,33 @@ constexpr uint32_t LMASK = 0x7fffffff;
 
 class mt19937_engine {
 public:
-  /**
-   * Note: This constructor is only used for testing this
-   * implementation against std::mt19937. If you use this
-   * constructor, chances are you'll encounter the same seeding
-   * surprises as mentioned in this article:
-   * http://www.pcg-random.org/posts/cpp-seeding-surprises.html
-   */
+
```

Comment:
What happened to this comment?

Reply:
This comment will come back once @fritzo and @neerajprad gets a chance to investigate why changing initialization scheme of mt19937 breaks test_distributions.py